docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -71,7 +71,13 @@ Go::
 -- 
 [source,go]
 ----
-
+func employee(name string, surname string) serialization.JSON {
```

Comment:
Alternative way to do this I think is the following:

```go
type Employee struct {
	Name string `json:"name"`
	Surname  string  `json:"surname"`
}
// To get a JSON value: 
b, err := json.Marshal(Employee{Name: "a", Surname: "b"})
if err != nil {
     jv := serialization.JSON(b)
     // use JSON value 
}
```

and this may be easier for larger structs. WDYT? Should we switch? cc @yuce

Reply:
We discussed this alternative with @yuce but decided not to make things complex and use a simple function. We thought that we don't have to strictly follow the same structure with Java, wdyt?